pkg/cmd/roachtest/test_runner.go

```diff
@@ -1786,3 +1794,18 @@ func testTimeout(spec *registry.TestSpec) time.Duration {
 	}
 	return timeout
 }
+
+// Annotate the start of the test in Grafana and the branch if applicable.
+func grafanaAnnotateTestStart(ctx context.Context, t test.Test, c cluster.Cluster) {
+	const BuildBranch = "TC_BUILD_BRANCH"
+	text := fmt.Sprintf("Starting %s", t.Name())
+	var tags []string
+	branch := os.Getenv(BuildBranch)
+	if branch != "" {
+		tags = []string{branch}
+	}
+
+	if err := c.AddGrafanaAnnotation(ctx, t.L(), grafana.AddAnnotationRequest{Text: text, Tags: tags}); err != nil {
+		t.L().Printf(err.Error())
```

Comment:
I'm fine adding this, but do you think the confusion would be from people not realizing

1. It's a Grafana Annotation error in general?
2. That Grafana Annotation error = not important?

If it's 1, maybe a more generic approach of wrapping all `c.AddGrafanaAnnotation` returns with `Error adding Grafana Annotation` would suffice?

If it's 2, then maybe we need to add a `WARNING:` in front of the error (or maybe something even less strongly worded?)

Reply:
Maybe a combination of 1 and 2? I'm not sure every `err.Error()` string will be identifiable enough (now and moving forward) that a reader can easily say "this is coming from a call to add a grafana annotation to the test-eng grafana instance".

As for 2), I think the context that the annotation is about test start/end helps understand the consequences. If the test also adds its own grafana annotations, it might not be clear which annotation the error is about.

I might be exaggerating the magnitude of the confusion this adds and if you think the current approach is fine, I'm onboard. My perspective is in trying to reduce (although minimally) the amount of unrecognizable logging we have, especially in "public" log files such as `test.log`.

Reply:
SGTM, thanks for the feedback, added both your original suggestion and a more generic `Error adding Grafana Annotation` prefix.

Reply:
Thanks!